numpy/linalg/linalg.py

```diff
@@ -1382,7 +1382,7 @@ def cond(x, p=None):
 
     Parameters
     ----------
-    x : (M, N) array_like
+    x : (..., M, N) array_like
```

Comment:
Yes it's true that part of the implementation of `cond` uses `svd` and that `svd` works on non-square matrices. But `cond` is the condition number of a particular function (matrix inversion, or `inv`) with respect to x, and it does not make sense to me that the condition number would be defined for x when x is not in the domain of the function.
https://en.wikipedia.org/wiki/Condition_number